internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -55,23 +55,30 @@ type AutoScalingConfig struct {
 	TargetMemory float64 `yaml:"targetMemory"`
 }
 
+// LBFargateManifestProps contains all properties for creating a new manifest.
+type LBFargateManifestProps struct {
+	AppName    string
```

Comment:
Do you think app_manifest.go should have an `AppManifestProps` struct with just AppName (for now) and compose this struct with that? I'm thinking of when we start adding new pattern types, there'll be some shared fields.

Reply:
yeah we can do that